grammarinator/generate.py

```diff
@@ -242,6 +242,9 @@ def restricted_float(value):
     parser.add_argument('-c', '--cooldown', default=1.0, type=restricted_float, metavar='NUM',
                         help='cool-down factor defines how much the probability of an alternative should decrease '
                              'after it has been chosen (interval: (0, 1]; default: %(default)f).')
+    parser.add_argument('-s', '--seed', type=int, metavar='NUM',
```

Comment:
Please, rename this to '--random-seed' and drop the single-letter command line option variant.

Also, this should be moved to the "Auxiliary settings" group after the '--sys-recursion-limit' argument.